packages/orbit-components/.storybook/preview.ts

```diff
@@ -29,6 +29,9 @@ const preview: Preview = {
   decorators: [orbitDecorator],
   parameters: {
     viewport: { viewports },
+    a11y: {
```

Comment:
@sarkaaa Could you please create a separate PR with this configuration and the changes on `orbitDecorator.tsx`? 🙏  I think that part can be extracted and merged already :)

Reply:
Done - https://github.com/kiwicom/orbit/pull/4510

Reply:
thanks 🫶 
resolving